pkg/util/exec/exec.go

```diff
@@ -38,7 +38,8 @@ import (
 const TimeoutWaitingForMessage = "exec timeout waiting for"
 
 var (
-	CephCommandsTimeout = 15 * time.Second
+	CephCommandsTimeout         = 15 * time.Second
+	RadosgwAdminCommandsTimeout = 20 * time.Second
```

Comment:
How about a separate PR for increasing the timeout? It's unrelated to the error handling. And do you have more background for how the timeout increase will help?